libs/wingsdk/src/core/app.ts

```diff
@@ -47,6 +47,11 @@ export interface AppProps {
    * @default false
    */
   readonly isTestEnvironment?: boolean;
+
+  /**
+   * App's external context
```

Comment:
What does this mean? (add more description)

Reply:
It just means that I experimented with a different property before and forgot to update the docstring, thanks for noticing!